src/Storefront/Resources/views/storefront/page/product-detail/review/review.html.twig

```diff
@@ -218,7 +218,7 @@
                                                 {% block page_product_detail_review_list_paging_form %}
                                                     <div class="product-detail-review-pagination">
                                                         <form class="product-detail-review-pagination-form"
-                                                              action="{{ path('frontend.product.reviews', { productId: reviews.productId }) }}"
+                                                              action="{{ path('frontend.product.reviews', { productId: reviews.productId, parentId: reviews.parentId }) }}"
```

Comment:
Don't you need to make the same fix in the `src/Storefront/Resources/views/storefront/component/review/review.html.twig` template? 
At least you did that here: https://github.com/shopware/platform/pull/2389

Reply:
@keulinho yes, you are absolutely right. I actually wanted to include this change but somehow forgot it... Thanks :)